.eslintrc.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
 		'jest/expect-expect': ['error', { assertFunctionNames: ['expect', 'expectTypeOf', 'verify'] }],
 
 		'@typescript-eslint/ban-types': 'off',
-		'@typescript-eslint/ban-ts-comment': ['warn', { 'ts-expect-error': false }],
+		'@typescript-eslint/ban-ts-comment': 'off', // covered by prefer-ts-expect-error
```

Comment:
I think `'@typescript-eslint/ban-ts-comment'` checks more things than just `@ts-ignore` and `@ts-expect-error`, doesn't it?

Reply:
You're right. I've changed this to allow ts-ignore and ts-expect-error only with explanations. Even though ts-ignore is whitelisted, prefer-ts-expect-error will still complain if you try to use it, so to use ts-ignore you need an explanation _and_ a eslint-disable-next-line. Which is a good thing IMO, ts-ignore is dangerous and should only be used rarely!